--- homebrew.py	2021-11-05 10:13:22.000000000 +0000
+++ homebrew_patched.py	2021-11-05 10:13:59.000000000 +0000
@@ -473,18 +473,12 @@
 
         cmd = [
             "{brew_path}".format(brew_path=self.brew_path),
-            "info",
+            "list",
             self.current_package,
         ]
         rc, out, err = self.module.run_command(cmd)
-        for line in out.split('\n'):
-            if (
-                re.search(r'Built from source', line)
-                or re.search(r'Poured from bottle', line)
-            ):
-                return True
 
-        return False
+        return rc != 0
 
     def _current_package_is_outdated(self):
         if not self.valid_package(self.current_package):
